apps/speed.c

```diff
@@ -1665,103 +2032,117 @@ int speed_main(int argc, char **argv)
 
     for (; *argv; argv++) {
         const char *algo = *argv;
+	int algo_found = 0;
```

Comment:
I suppose the locate functions could've been moved to the top of the loop, and the the `continue`s would be able to remain... but other stuff would need to be done as well... so this can be left. It's not in a critical part of the code.